admin_bill_of_rights.md

```diff
@@ -12,13 +12,13 @@ Therefore, we hold the following truths to be self-evident.
 
 Administrators have the right to:
 
-1. [Centrally administer](#centrally_admin) their organizations’ cloud resources.
+1. [Centrally administer](#centrally_admi) their organizations’ cloud resources.
```

Comment:
Are you sure this is correct? 🤔

Reply:
Hey @magnologan this was another intentional typo from the testing in my fork. Thanks for catching! I have reverted for correctness.